packages/wonder-blocks-button/src/__docs__/best-practices.stories.mdx

```diff
@@ -0,0 +1,107 @@
+import {Meta, Story, Canvas} from "@storybook/addon-docs";
+import {StyleSheet} from "aphrodite";
+
+import Button from "@khanacademy/wonder-blocks-button";
+import {View} from "@khanacademy/wonder-blocks-core";
+
+<Meta
+    title="Navigation/Button/Best practices"
+    component={Button}
+    parameters={{
+        previewTabs: {
+            canvas: {hidden: true},
+        },
+        viewMode: "docs",
+        chromatic: {
+            // Disables chromatic testing for these stories.
+            disableSnapshot: true,
+        },
+    }}
+/>
+
+## Best Practices
+
+### Layout
+
+In vertical layouts, buttons will stretch horizontally to fill the available
+space. This is probably not what you want unless you're on a very narrow
+screen.
+
+<Canvas>
+    <Story name="Full-bleed button">
+        <View>
+            <Button>Label</Button>
+        </View>
+    </Story>
+</Canvas>
+
+This can be corrected by applying appropriate flex styles to the container.
+
+<Canvas>
+    <Story name="Buttons in rows">
+        <View>
+            <View style={styles.row}>
+                <Button>Button in a row</Button>
+            </View>
+            <View style={styles.gap} />
+            <View style={styles.column}>
+                <Button>Button in a column</Button>
+            </View>
+        </View>
+    </Story>
+</Canvas>
+
+### Usign minWidth for internationalization
+
```

Comment:
I wish we had a way to lint for this.

Reply:
Agreed! That's why I tried to keep the main stories on the `js` side. I think the main benefit of MDX would be for these kind of informative pages. I'll try to create a page in Confluence to document all these thoughts/gotchas.